static/js/ui.js

```diff
@@ -566,6 +566,26 @@ $(function () {
 
     $('.message_failed i[data-toggle="tooltip"]').tooltip();
 
+    $('.copy_message[data-toggle="tooltip"]').tooltip({ trigger: 'manual',
+                                       animation: false });
+
+    $("body").on("mouseover", "#message_edit_content", function () {
+        $(this).prev().prev().show();
+    });
+
+    $("body").on("mouseout", "#message_edit_content", function () {
+        $(this).prev().prev().hide();
+    });
```

Comment:
This code toggles the show/hide of the button when the mouse hovers over the textarea